routes/api.php

```diff
@@ -17,6 +17,7 @@
 /* Read */
 Route::group(['middleware' => ['auth:sanctum', 'ability:' . TokenEnum::READ->value]], function () {
     Route::get('/products/location', [ProductController::class, 'getProductsByUserLocation']);
+    Route::post('/roles/get_roles', [RoleController::class, 'getRoles']);
```

Comment:
Send bearer token as header to use get instead of post request.

Reply:
Will do :)